packages/jwt-verifier/README.md

```diff
@@ -131,6 +131,18 @@ const oktaJwtVerifier = new OktaJwtVerifier({
 });
 ```
 
+## Testing
+Setup SPA and Web App in your Okta org and testing environment variables by following [Testing](https://github.com/okta/okta-oidc-js#testing) section in Monorepo's README.
```

Comment:
Would it be more clear to say 'okta-oidc-js Monorepo's README'?

Reply:
Updated